lib/common/resources/otel-collector-config.yml

```diff
@@ -55,6 +55,31 @@ spec:
                 regex: (.+)
                 target_label: __metrics_path__
                 replacement: /api/v1/nodes/$${1}/proxy/metrics/cadvisor
+            - job_name: 'apiserver'
```

Comment:
You still need to have annotation to set and unset like how you see in blueprints